app/views/questions/_answer.html.erb

```diff
@@ -43,8 +43,17 @@
           <a href="/answers/accept/<%= answer.id %>" data-remote="true" id="answer-<%= answer.id %>-accept" class="answer-accept btn btn-sm btn-default">Accept this answer</a>  
         <% end %>
       <% end %>
+    <% if answer.likers.length != 0 %>
+        <div class="btn btn-default btn-sm" rel="popover" data-placement="left" data-html="true" data-title="Users who like this" data-content="
+            <% answer.likers.each do |user| %>
```

Comment:
@jywarren If we do go ahead with the modal, I think it makes sense to incorporate it into the 'notes' liked menu as well.
@ryzokuken How about opening up another issue?